### DISCUSSION TITLE
A way to link checksums of different consoles

### DISCUSSION THREAD

**Discussion by Alfex3** [RetroAchievements/RAWeb#793] [Ideas]
The Game Boy Advance version of Zelda II: The Adventure of Link had its set demoted, and the ROM was linked to the SNES entry. Currently this is something only admins can do, so there should be a way to link a game to an entry of a different console (example: the PS1 and N64 versions of Toy Story 2 are the same), either by making the dropdown show all the games, not just the ones of that console, or by giving the option to add a checksum by copying and pasting.

**Comment by Jamiras** [comment]
Even if the alternate system is emulating the original ROM, it's unlikely that the memory addresses will line up, so all the achievements would have to be rewritten for the alternate version.

**Comment by Alfex3** [comment]
That's what I've been doing for a long time when adding region support, and what was also done with Zelda II. Someone might not know about the Zelda II case and develop another set for a game that's exactly the same.

I don't know if the author of the Phoenix Wright: Ace Attorney set knew that the GBA version had a set. The GBA set might be demoted in the future, noting that the achievements for the 5th case are earnable only in the DS version
